lib/utils/Kconfig

```diff
@@ -12,9 +12,9 @@ config JSON_LIBRARY
 config RING_BUFFER
 	bool "Ring buffers"
 	help
-	  Enable usage of ring buffers. This is similar to kernel FIFOs but ring
-	  buffers manage their own buffer memory and can store arbitrary data.
-	  For optimal performance, use buffer sizes that are a power of 2.
+	  Provide a highly efficient ring buffer management facility for
+	  arbitrary data. Kernel FIFOs are built on top of this. May be used
```

Comment:
When I think of kernel FIFOs, I think of struct k_fifo, which is built atop k_queue, which in turn is built atop sys_sflist_t. I suspect you are referring to something else, so this should probably get reworded.

Reply:
Yeah, I meant kernel pipes. Will reword.